changelog.md

```diff
@@ -10,6 +10,18 @@ _Nothing yet.._
 - react-native-bpk-component-icon: 1.1.15 => 1.2.0
  - `icons` object is now exposed, allowing users to refer to icon names using a constant.
 
+**Breaking:**
+- bpk-component-banner-alert:
+  - Removed `dismissable` property in favour of `configuration`.
+  - A banner alert with `children` will no longer be auotmatically expandable. Use the newly introduce `configuration` prop instead.
+
```

Comment:
You've also removed `ARIA_LIVE` stuff...

Reply:
Good eye